docs/codeql/codeql-cli/creating-codeql-query-suites.rst

```diff
@@ -118,8 +118,10 @@ typically a query metadata property. The value can be:
 
 To match a constraint, a metadata value must match one of the strings or
 regular expressions. When there is more than one metadata key, each key must be matched.
-For more information about query metadata properties, see ":ref:`Metadata for CodeQL queries
-<metadata-for-codeql-queries>`."
+The standard metadata keys available to match on are: ``description``, ``id``, ``kind``,
+``name``, ``tags``, ``precision``, ``problem.severity``, and ``security-severity``.
```

Comment:
Should we omit `security-severity` for now, since you can't yet filter on it with inequality constraints?